vignettes/userguide.Rmd

```diff
@@ -189,6 +189,42 @@ options(
 
 We recommend that you [set up SSH keys](https://www.digitalocean.com/community/tutorials/how-to-configure-ssh-key-based-authentication-on-a-linux-server) for password-less login.
 
+### Options
+
+The various configurable options are mentioned throughout the documentation where applicable, however, we list all of the options here for reference.
```

Comment:
Please keep line length to 80 chrs to be consistent with the rest of the document